TabulateSmarterTestPackage/TabulateSmarterTestPackage.csproj

```diff
@@ -48,6 +45,30 @@
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)|$(RuntimeIdentifier)' == 'Release|linux-x64' ">
```

Comment:
I'm a bit fuzzy on using external publish configs and embedding it. This is here because `dotnet` didn't see it otherwise. The `*.pubxml` files got put there by VS. I'm pretty sure one of these should go away.

Reply:
I'm not 100% sure about the external publish vs. embedding either, but in my limited reading of it, the embedding seems to be tied to publishing through VS, whereas the external file allows for command line publishing.